testomania/src/main/java/com/earth/testomania/home_screen/presentation/HomeScreen.kt

```diff
@@ -183,13 +199,16 @@ fun CardButton(
     navigator: DestinationsNavigator? = null,
     scaffoldState: ScaffoldState,
     modalBottomSheetState: ModalBottomSheetState,
-) {
+    networkConnectivityObserver: NetworkConnectivityObserver,
+    ) {
     val scope = rememberCoroutineScope()
     val comingSoonStr = stringResource(id = R.string.coming_soon)
     val dismissStr = stringResource(id = R.string.dismiss)
     val viewModel: HomeScreenViewModel = hiltViewModel()
+    val status by networkConnectivityObserver.observe()
```

Comment:
It's cool that you are blocking buttons if there is no connection, but we need to allow the About button regarding of connection status.
Let's give something for fun and the user can read our About at least if the user does not have a connection :D

Reply:
Based on this issue https://github.com/Nodrex/Testomania/issues/266 We make the About screen dependent on the Internet.